src/test/eo/org/eolang/math/number-tests.eo

```diff
@@ -979,3 +979,63 @@
     ln.
       number 20
     $.equal-to 2.995732273553991
```

Comment:
@levBagryansky can we guarantee that the answer will be exactly that? For now, yes, but in the future someone may change the implementation a bit and, for example, the last character will change. So I propose to see that the difference from the answer modulo is very small.